etc/workflows/failedRecording.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>failedrecording</id>
+  <title>Recording failed</title>
+  <tags>
+  <tag>editor</tag>
+  <tag>archive</tag>
+  </tags>
+  <description/>
+  <configuration_panel/>
+
+  <operations>
+
+    <!-- Resolve the comment for cutting -->
+    <operation
+      id="comment"
+      description="Resolve the cutting flag">
+      <configurations>
+        <configuration key="description">Recording has not been cut yet.</configuration>
+        <configuration key="reason">EVENTS.EVENTS.DETAILS.COMMENTS.REASONS.CUTTING</configuration>
+        <configuration key="action">resolve</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Untag elements for deletion -->
+    <operation
+      id="tag"
+      max-attempts="2"
+      fail-on-error="true"
+      exception-handler-workflow="error"
+      description="Tagging media package elements for deletion">
+      <configurations>
+        <configuration key="source-flavors">*/trimmed,*/source</configuration>
+        <configuration key="target-tags">-atom,-archive</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Copy placeholder -failed- Video for publication -->
+    <operation
+      id="compose"
+      fail-on-error="true"
+      exception-handler-workflow="error"
+      description="copy failed Video">
+      <configurations>
+        <configuration key="source-flavor">*/source</configuration>
+        <configuration key="target-flavor">presenter/delivery</configuration>
+        <configuration key="target-tags">failed,+archive,+engage-download</configuration>
+        <configuration key="encoding-profile">failed.http</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Archive the current state of the media package -->
+    <operation
+      id="snapshot"
+      fail-on-error="true"
+      exception-handler-workflow="partial-error"
+      description="Archiving">
+      <configurations>
+         <configuration key="source-flavors">dublincore/*,security/*,*/delivery</configuration>
+         <configuration key="source-tags">archive</configuration>
```

Comment:
```suggestion
         <configuration key="source-tags">failed</configuration>
```

Using the `failed` tag here would mean that you could remove the `tag` operation from above since you would not need to remove any tags.

Reply:
I don't get that. Maybe you can explain this. You need to tag the items you want to keep in the last snapshot. And untag the Items you want to delete.

Reply:
Okay, after our discussion. Here is the explanation. Maybe it could be done in a different way, but to keep it in  the most "common" state as other workflows do. I will stay witch the archive tag.